pkg/collector/python/init.go

```diff
@@ -228,9 +228,21 @@ func sendTelemetry(pythonVersion string) {
 func detectPythonLocation(pythonVersion string) {
 	// Since the install location can be set by the user on Windows we use relative import
 	if runtime.GOOS == "windows" {
-		_here, _ := executable.Folder()
+		_here, err := executable.Folder()
+		if err != nil {
+			log.Warnf("Error getting executable folder: %v", err)
+			log.Warnf("Trying again without symlinks")
+			_here, err = executable.FolderNoSymlinks()
```

Comment:
```suggestion
			_here, err = executable.FolderNoResolveSymlinks()
```
The name "NoSymlinks" made me think it resolved them (thus removing them), but instead this is the one that leaves them in. Maybe this name makes the intent more clear?